public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -140,6 +140,27 @@ export const getTimestampPrecision = (timestamp: number): 'millis' | 'micros' |
   }
 };
 
+export const appendModeToTraceViewUri = (
+  traceId: string,
+  getTraceViewUri?: (traceId: string) => string,
+  traceMode?: string | null
+): string => {
+  const baseUri = getTraceViewUri ? getTraceViewUri(traceId) : '';
+  const isHashRouter = baseUri.includes('#');
+  const separator = isHashRouter
+    ? baseUri.includes('?')
+      ? '&'
+      : '?'
+    : baseUri.includes('?')
+    ? '&'
+    : '?';
```

Comment:
nit. is this just `const separator = baseUri.includes('?') ? '&'
      : '?'

Reply:
Fixed thank you.